VerilogCodingStyle.md

```diff
@@ -751,7 +751,7 @@ operations unambiguous.
 
 #### Ternary Expressions
 
-***Nested ternary expressions must be enclosed in parentheses.***
+***It is recommended to enclose nested ternary expressions in parentheses.***
```

Comment:
Do we actually want to recommend it? I normally find that with nested ternary's you most often use the style with parentheses omitted below where you're building a priority mux. The ones where you do want parantheses are the rarer ones where the true condition of the ternary has a ternary itself:


```
assign foo = condition_a ? (condition_a_x ? x : y) : b;
```

Or blending the two:

```
assign foo = condition_a ? (condition_a_x ? x : y) :
             condition_b ? b                       :
                           not_a_nor_b; 
```

So it's not a case of recommended or not recommended but rather how you are nesting the ternarys that should guide the use of parentheses.